dbfit-java/core/src/main/java/dbfit/api/PreparedDbCommand.java

```diff
@@ -0,0 +1,18 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * Represents a prepared database statement with ability to bind input parameters and
+ * to register and access output ones including return values.
+ */
+public interface PreparedDbCommand extends DbCommand, DbQuery {
```

Comment:
So, perhaps we'd only have to know if we're creating an `INSERT` statement or not, but more specifically whether we, potentially, need to get at any generated keys and this would only be where we're building an `INSERT` statement for the `Insert` fixture I think.